homework10 fixed/schoolUI/schoolUI/ViewController.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  ViewController.swift
+//  Lesson 7 homework
+//
+//  Created by Ernest Mihasenko on 22.03.22.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    let school = School(students: [])
+ 
+    @IBOutlet weak var studentSurnameTextField: UITextField!
+    
+    @IBOutlet weak var studentNameTextField: UITextField!
+    
+    @IBOutlet weak var addStudentButton: UIButton!
+    
+    @IBOutlet weak var studentGradeTextField: UITextField!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+
+    }
+        
+    @IBAction func addStudent(_ sender: Any) {
+//        let school = School(students: studentsArray)
+//       let student = Student(name: studentNameTextField.text ?? "Имя не введено", surname: studentSurnameTextField.text ?? "Фамилия не введена", grade: studentGradeTextField.text ?? "Класс не введен")
+//       studentsArray.append(student)
+        
+        guard let studentName = studentNameTextField.text else {
+            return
+        }
+        guard let studentSurname = studentSurnameTextField.text else {
+            return
+        }
+        guard let studentGrade = studentGradeTextField.text else { return
+        }
```

Comment:
You should either have `}` on the same line as `return` or move `return` on a new line as you did above